docs/tutorial-traffic.md

```diff
@@ -0,0 +1,155 @@
+
+We see:
+
+* A single Pod, `po/echoheaders-2787888573-c3nt1`, which runs the echoserver container
+* A single Replica Set, `rs/echoheaders-2787888573`, which manages the Pod and restarts it if it ever fails
+* A single Deployment, `deploy/echoheaders`, which is the object that created the Replica Set. This can be edited to update the application without causing downtime.
+
+## 2\. Expose our application to the Internet
+
+Next we can use `kubectl expose` to create a Service that exposes our new application to the Internet over an AWS Elastic Load Balancer (ELB).
+
+Let's run the following command:
+    
+    
+    kubectl expose --namespace=simple-demo deployment echoheaders --type=LoadBalancer --port=80 --target-port=8080 --name=echoheaders-public
+    
+
+This will create a Service object, which acts as an endpoint for routing to an instance of the `echoheaders` application.
+
+After we expose the deployments, we can inspect the results with `kubectl describe`:
+    
+    
+    $ kubectl --namespace=simple-demo describe service echoheaders-public
+    Name:           echoheaders-public
+    Namespace:      simple-demo
+    Labels:         run=echoheaders
+    Selector:       run=echoheaders
+    Type:           LoadBalancer
+    IP:         10.103.66.255
+    LoadBalancer Ingress:   a9201a1bdfc6411e68fdc06048bde387-495139964.us-west-1.elb.amazonaws.com
+    Port:            80/TCP
+    NodePort:        30031/TCP
+    Endpoints:      192.168.96.196:8080
+    Session Affinity:   None
+    Events:
+      FirstSeen LastSeen    Count   From            SubObjectPath   Type        Reason          Message
+      --------- --------    -----   ----            -------------   --------    ------          -------
+      2m        2m      1   {service-controller }           Normal      CreatingLoadBalancer    Creating load balancer
+      1m        1m      1   {service-controller }           Normal      CreatedLoadBalancer Created load balancer
+    
+
+This describes the `echoheaders-public` service that was just created. Because it is `type=LoadBalancer`, we have a public-facing URL under the `LoadBalancer Ingress` section. We can now browse to the resulting URL and see our application:
+
+![../../_images/echoheaders-screenshot.png][7]
+
+The port for the service is 80, which is the public external port for this service. But the endpoint is on port 8080, which is the port the container itself is listening on.
+
+For more information about Services, see the [Kubernetes User Guide reference for Services][8].
+
+## Reference: The kubectl expose command
+    
+    
+    NAMESPACE=simple-demo
+    RESOURCETYPE=deployment
+    RESOURCENAME=echoheaders
+    SERVICETYPE=LoadBalancer
+    PORT=80
+    TARGETPORT=8080
+    SERVICENAME=echoheaders-public
+    
+    kubectl expose --namespace=$NAMESPACE 
+    $RESOURCETYPE $RESOURCENAME 
+    --type=$SERVICETYPE 
+    --port=$PORT --target-port=$TARGETPORT 
+    --name=$SERVICENAME
+    
+
+Parameters:
+
+| ----- |
```

Comment:
These tables seem properly formatted and markdown viewers show them correctly. However when I go to my fork and just view the page it is not rendering a table.

Reply:
Fixed; I guess the table format changed at some point. Has to start with the headers and not a top separator.